project.clj

```diff
@@ -1,8 +1,7 @@
-(defproject google-civic "2.0.2-SNAPSHOT"
+(defproject google-civic "2.1.0-SNAPSHOT"
   :description "Google Civic Info API client"
   :dependencies [[org.clojure/clojure "1.7.0"]
-                 [clj-http "2.0.0"]
-                 [cheshire "5.5.0"]
-                 [slingshot "0.12.2"]]
+                 [clj-http "2.3.0"]
```

Comment:
Cool...would be nice to know if there's a big value proposition to moving to 3.x as well. Being faster might or might not be a good thing at this point, if we're particularly trying to add brakes for when it goes too fast for Google.